.circleci/config.yml

```diff
@@ -11,6 +11,12 @@ jobs:
 
     steps:
       - checkout
+      - run:
+          name: Check for large files
+          command: |
+            set +e
+            find . -size +1000000c -not -path './.git/*' | grep .
+            (( $? == 1 ))
```

Comment:
`find` will output matching files. `grep .` will match any file, so if `find` outputs any files, `grep` will return with an exit status of 0. We want the opposite though: we want there to be no matching files. In that case, `grep` returns with an exit status of 1. In other words, we want the command as a whole to succeed only if `grep` returns 1.

It [looks like](https://circleci.com/docs/configuration-reference/#default-shell-options) CircleCI runs the command with `-e`, which would cause the command to fail if `grep` returned with an exit status of 1. That's why I run `set +e`.

Reply:
`(( $? == 1))` seems unidiomatic compared to e.g. `[[ $? -eq 1]]`.

An alternative to using `grep` might be:

```sh
find . -size +1000000c -not -path './.git/*' \
  -execdir bash -c '(echo "Big file(s) found: $@" && exit 1)' -- {} + ;
```

Reply:
I've replaced `[[ ... ]]` with `(( ... ))`. :+1:

I feel less sure about `-execdir bash ...`. At least to me, it doesn't feel much clearer than `grep`.

Reply:
`+1M` looks (roughly) equivalent to `+1000000c`, and may be easier to read.

Reply:
That was my first choice as well and worked for me locally. You can see `+1Mc` in the first commit above. However, I saw this message in CircleCI:

> ```
> find: Invalid argument `+1Mc' to -size
> ```